src/mpris.rs

```diff
@@ -463,11 +463,14 @@ impl MprisPlayer {
 }
 
 /// Commands to control the [MprisManager] worker thread.
+#[derive(Debug)]
 pub enum MprisCommand {
-    /// Notify about playback status and metadata updates.
-    NotifyPlaybackUpdate,
-    /// Notify about volume updates.
-    NotifyVolumeUpdate,
+    /// Emit playback status
+    Playback,
+    /// Emit volume
+    Volume,
+    /// Emit metadata
+    Metadata,
```

Comment:
Right now the naming is not very clear as it's neither a command name although the enum type indicates a command-like naming.

Could you either change the enum to something like `MprisUpdateCommand`/`MprisEmitCommand` or have the single fields indicate something similar in the prefix/suffix? WDYT?

Reply:
Thanks for the review! This might have been the result of a careless ```sed``` command to fix clippy complaints. Sorry, this should not have slipped through, and was pretty careless of me.
What do you think about ```EmitPlaybackStatus``` etc.? I find that a bit clearer than, ```NotifyPlaybackUpdate``` since basically just send a message into the void and do not "update" anything.

Reply:
No worries! I'm fine with either, as you prefer :)